wandb/sdk/internal/system/system_monitor.py

```diff
@@ -44,9 +43,9 @@ def __init__(
         settings: "SettingsStatic",
         interface: "Interface",
     ) -> None:
-
-        self._shutdown_event: mp.synchronize.Event = mp.Event()
-        self._process: Optional[Union[mp.Process, threading.Thread]] = None
+        # todo:
```

Comment:
What is this todo?

Reply:
oops, forgot to push the clean up the rm... dunno :)